sysutils/cfiles/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        mananapr cfiles 1.7.1 v
+categories          sysutils
+platforms           darwin
+license             MIT
+maintainers         {@sikmir gmail.com:sikmir} openmaintainer
+
+description         A ncurses file manager written in C with vim like keybindings
+long_description    ${name} is a terminal file manager with vim like keybindings, \
+                    written in C using the ncurses library. It aims to provide an \
+                    interface like ranger while being lightweight, fast and minimal.
+
+checksums           rmd160  263993e1f7f9813753a208e4a86d9fb0dda65013 \
+                    sha256  10d3ca8d167c0c74c41d9213933e85519d75a302fba54614682c13381a9d93ad \
+                    size    237807
+
+patchfiles          patch-config.h.diff \
+                    patch-Makefile.diff
+
+use_configure       no
+depends_build       port:pkgconfig
+depends_lib         port:ncurses
+depends_run         path:bin/pdftoppm:poppler \
+                    port:atool \
+                    port:fzf \
+                    port:mediainfo
+
+build.args-append   CC=${configure.cc}
+
+post-patch {
+    reinplace "s#/usr/bin#${prefix}/share/cfiles/scripts#g" ${worksrcpath}/config.h
+}
+
+post-build {
+    system "cd ${worksrcpath} && gzip cfiles.1"
```

Comment:
We don't like the use of cd in system commands. Change this line to:

`system -W ${worksrcpath} "gzip cfiles.1"`

Reply:
Fixed